model/user.go

```diff
@@ -16,24 +16,25 @@ import (
 )
 
 const (
-	ME                             = "me"
-	USER_NOTIFY_ALL                = "all"
-	USER_NOTIFY_MENTION            = "mention"
-	USER_NOTIFY_NONE               = "none"
-	DESKTOP_NOTIFY_PROP            = "desktop"
-	DESKTOP_SOUND_NOTIFY_PROP      = "desktop_sound"
-	MARK_UNREAD_NOTIFY_PROP        = "mark_unread"
-	PUSH_NOTIFY_PROP               = "push"
-	PUSH_STATUS_NOTIFY_PROP        = "push_status"
-	EMAIL_NOTIFY_PROP              = "email"
-	MOBILE_NOTIFY_PROP             = "mobile"
-	MOBILE_PUSH_STATUS_NOTIFY_PROP = "mobile_push_status"
-	CHANNEL_MENTIONS_NOTIFY_PROP   = "channel"
-	COMMENTS_NOTIFY_PROP           = "comments"
-	MENTION_KEYS_NOTIFY_PROP       = "mention_keys"
-	COMMENTS_NOTIFY_NEVER          = "never"
-	COMMENTS_NOTIFY_ROOT           = "root"
-	COMMENTS_NOTIFY_ANY            = "any"
+	ME                                  = "me"
+	USER_NOTIFY_ALL                     = "all"
+	USER_NOTIFY_MENTION                 = "mention"
+	USER_NOTIFY_NONE                    = "none"
+	DESKTOP_NOTIFY_PROP                 = "desktop"
+	DESKTOP_SOUND_NOTIFY_PROP           = "desktop_sound"
+	MARK_UNREAD_NOTIFY_PROP             = "mark_unread"
+	PUSH_NOTIFY_PROP                    = "push"
+	PUSH_STATUS_NOTIFY_PROP             = "push_status"
+	EMAIL_NOTIFY_PROP                   = "email"
+	MOBILE_NOTIFY_PROP                  = "mobile"
+	MOBILE_PUSH_STATUS_NOTIFY_PROP      = "mobile_push_status"
+	CHANNEL_MENTIONS_NOTIFY_PROP        = "channel"
+	COMMENTS_NOTIFY_PROP                = "comments"
+	MENTION_KEYS_NOTIFY_PROP            = "mention_keys"
+	COMMENTS_NOTIFY_NEVER               = "never"
+	COMMENTS_NOTIFY_ROOT                = "root"
+	COMMENTS_NOTIFY_ANY                 = "any"
+	IGNORE_CHANNEL_MENTIONS_NOTIFY_PROP = "ignore_channel_mentions"
```

Comment:
Can you keep this with the other constants for the setting? I don't think we'll ever have this in the user's account-wide NotifyProps because it already has `CHANNEL_MENTIONS_NOTIFY_PROP` for it